Sources/App/routes.swift

```diff
@@ -101,6 +101,9 @@ func routes(_ app: Application) throws {
             app.get(":owner", ":repository", ":reference", "js", "**") {
                 try await PackageController.documentation(req: $0, fragment: .js)
             }.excludeFromOpenAPI()
+            app.get(":owner", ":repository", ":reference", .fragment(.linkableEntities)) {
```

Comment:
This is not strictly necessary, as it only allows SPI to serve the `linkable-entities.json` which is not necessary for us to fetch it for parsing. However, I added it because I bet we'll want to check what's in it occasionally and this is easier than figuring out the S3 URL.

We might want to add it here and remove it later, or maybe just remove it now.

Reply:
My gut feeling is not to serve it unless it serves an actual purpose for the DocC app. This way there's no doubt which parts it needs and which parts it doesn't, and we're not tempted to run any lower level things through the proxy when we should really be going to the source (say "fetch all `linkable-entities.json` files").

Reply:
So I just started removing this and it's a little tricky. We use the same fragment to build the AWS URLs for fetching resources directly from S3. If we don't have a `linkableEntities` in the enum, we can't easily build the URL we need to fetch the file.

What I'd suggest is, given that this is already a large PR, we don't tackle solving this here but in a new issue afterwards.

Of course, I could be missing something and there may be an easy way to build the URL without a fragment.

Reply:
I think in that case we leave it, as it's become a required part of what's exposed from the S3 doc archive content. It's not the DocC app per se, but what _we_ use in the DocC-Proxy to host the DocC app.

What I was mainly getting at is that we should be mindful of which parts of the doc archive content we treat as opaque and which parts we explicitly reference.